packages/ic-asset-certification/src/asset_router.rs

```diff
@@ -908,10 +908,15 @@ impl<'content> AssetRouter<'content> {
                 http::header::CONTENT_RANGE.to_string(),
                 format!("bytes {range_begin}-{range_end}/{total_length}"),
             ));
-            request_headers.push((
-                http::header::RANGE.to_string(),
-                format!("bytes={range_begin}-"),
-            ));
+
```

Comment:
This is the main change. We don't want to add the range header to certification for the first request because the gateway doesn't send it, otherwise this request/response pair will fail response verification.